packages/contracts/docs/osx/01-how-it-works/01-core/02-permissions/01-conditions.md

```diff
@@ -173,7 +173,7 @@ In another use-case, we might want to make sure that the `sendCoins` function ca
 ```solidity title="PriceOracleCondition.sol"
 import '@chainlink/contracts/src/v0.8/interfaces/AggregatorV3Interface.sol';
 
-contract PriceOracleCondition is IPermissionCondition {
+contract PriceOracleCondition is PermissionConditionx {
```

Comment:
PermissionCondition(x) why this x?, if it is for display reasons, feel free to resolve this comment.

Reply:
no, was a typo indeed